packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -1,1114 +1,14 @@
-const React                 = require('react');
```

Comment:
if we go ahead with this approach, when rebasing this, should use a git move  here to preserve the history